test/check-application

```diff
@@ -1313,6 +1313,11 @@ class TestApplication(testlib.MachineCase):
 
         # CPU shares work only with system containers
         if auth:
+            # Check that the checkbox is enabled when clicked on the field
+            b.click('#run-image-cpu-priority')
+            b.wait_visible("#run-image-dialog-cpu-priority-checkbox:checked")
+            b.set_checked("#run-image-dialog-cpu-priority-checkbox", False)
+
```

Comment:
We should check that it is actually disabled before, so we are sure it did something. And I don't think we need to then unset it and set it back, surely there is some different place that checks these checkboxes.

Reply:
This is the only place the cpu priority is tested, so I'll leave the unsetting in.